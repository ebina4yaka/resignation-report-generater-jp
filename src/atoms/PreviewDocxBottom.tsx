import React from 'react'
import { createStyles, makeStyles } from '@material-ui/core/styles'

type Props = {
  dateOfNotification: string
  department: string
  name: string
}

const useStyles = makeStyles(
  createStyles({
    root: {
      textAlign: 'left',
    },
  })
)

export default function PreviewDocxBottom(props: Props): React.ReactElement {
  const { dateOfNotification, department, name } = props
  const classes = useStyles()
  return (
    <div className={classes.root}>
      <div>{dateOfNotification}</div>
      <div>{department}</div>
      <div>{name}</div>
    </div>
  )
}
