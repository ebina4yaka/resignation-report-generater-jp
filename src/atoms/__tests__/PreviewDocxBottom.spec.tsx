import renderer from 'react-test-renderer'
import PreviewDocxBottom from '../PreviewDocxBottom'

test('Snapshot test for PreviewDocxBottom', () => {
  const tree = renderer
    .create(
      <PreviewDocxBottom
        name="無職太郎"
        dateOfNotification="2020年11月7日"
        department="退職部"
      />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
