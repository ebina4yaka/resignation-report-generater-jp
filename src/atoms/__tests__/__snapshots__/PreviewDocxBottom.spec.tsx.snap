// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for PreviewDocxBottom 1`] = `
<div
  className="makeStyles-root-1"
>
  <div>
    2020年11月7日
  </div>
  <div>
    退職部
  </div>
  <div>
    無職太郎
  </div>
</div>
`;
